test/e2e/node/apparmor.go

```diff
@@ -28,8 +28,8 @@ var _ = SIGDescribe("AppArmor", func() {
 
 	ginkgo.Context("load AppArmor profiles", func() {
 		ginkgo.BeforeEach(func() {
-			common.SkipIfAppArmorNotSupported()
-			common.LoadAppArmorProfiles(f)
+			e2esecurity.SkipIfAppArmorNotSupported()
```

Comment:
can't comment on the file where this is defined but could you move the definition of `SkipIfAppArmorNotSupported` to https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/skip.go ?
we wan't to prevent calling the e2e test framework from within its subpkgs (i.e., test/e2e/-framework/security).

Reply:
done, thanks